packages/contracts/src/interfaces/Types.sol

```diff
@@ -90,7 +93,7 @@ struct TokenAllowance {
 struct ExecutionContext {
     address walletAddr; // Wallet address of the user
     address extensionAddr; // Address of extension in use
-    bool unclaimedFundRegistered; // Flag to indicate whether the unclaimed state has been registered
+    // bool unclaimedFundRegistered; // Flag to indicate whether the unclaimed state has been registered
```

Comment:
If the unclaimedFundRegistered is enabled when the extension registers an unclaimed fund, the sender needs to pay an extra fee in handleEmailOp. However, we assume the fee to register the unclaimed fund is paid not by the sender but by the extension contract. If the unclaimedFundRegistered is kept false when the extension registers it, I agree with using unclaimedFundRegistered.

Reply:
Sorry I didnt understand. Extension dont register unclaimed funds by calling the internal function 🤔

Reply:
Sorry, I misunderstood that unclaimedFundRegistered is enabled in the `registerUnclaimedFund`.
If this is enabled only in `_executeEmailOp` when the command is `send`, it has no problem.

Reply:
> If this is enabled only in _executeEmailOp when the command is send, it has no problem.

Yes, it is like this. So we can keep it in the context for cleaner code. I will revert this change?

Reply:
So there was I a bug - I was setting `currContext.unclaimedStateRegistered = true` in _execute (for wallet command) instead of `currContext.unclaimedFundRegistered = true` - maybe this confused you.